src/features/auth/sign-in/ui/sign-in-form.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import { useState } from 'react';
+import { useTranslations } from 'next-intl';
+import { useRouter } from '@/shared/i18n/navigation';
+import { buildSignInRules } from '../model/shema';
+import { useSignIn } from '../model/use-sign-in';
+import { mapSignInError } from '@/shared/api/firebase/map-sign-in-errors';
+
+import { Button, Form, Input, Typography } from 'antd';
+
+const { Item } = Form;
+
+type FieldType = {
+  email?: string;
+  password?: string;
+};
+
+export function SignInForm() {
+  const t = useTranslations('SignInForm');
+  const [form] = Form.useForm<FieldType>();
+  const rules = buildSignInRules(t);
+  const router = useRouter();
+
+  const { mutate, loading } = useSignIn();
+  const [apiError, setApiError] = useState<string | null>(null);
+
+  const onFinish = async (values: FieldType) => {
+    try {
+      setApiError(null);
+      await mutate({
+        email: values.email!,
+        password: values.password!,
+      });
```

Comment:
This is what I wrote to you about in another file.
I don't understand why you can't use it right away like this.

```suggestion
      await apiSignIn({
        email: values.email!,
        password: values.password!,
      });
```